src/AWSMetadataUtilities.jl

```diff
@@ -184,6 +184,44 @@ function _documentation_cleaning(documentation::String)
     return documentation
 end
 
+"""
+    _uri_cleaning(uri::String)
+
+Replace URI parameters with the appropriate syntax for Julia interpolation.
+
+Find all URI parameters, and apply the following replacements:
+* { => \$(
+* } => )
+* - => _
+* + => nothing
+
+Example:
+"/v1/configurations/{configuration-id}" => "/v1/configurations/\$(configuration_id)"
+
+# Arguments
+- `uri::String`: URI to be cleaned
+
+# Returns
+- `String`: Cleaned URI
+"""
+function _uri_cleaning(uri::String)
```

Comment:
```suggestion
function _clean_uri(uri::String)
```
Best practice for function naming is verbs